templates/part.card.php

```diff
@@ -69,7 +69,7 @@
 
 		</h3>
 		<textarea elastic ng-if="status.cardEditDescription"
-				  placeholder="Enter your description here…"
+				  placeholder="<?php p($l->t('Add a card description…')); ?>"
```

Comment:
Just a detail, but we use a space between the last word and the ellipsis always. :)